docs/docs/gatsby-cli.md

```diff
@@ -41,12 +38,18 @@ development server.
 | `-o`, `--open`  | Open the site in your (default) browser for you |
 | `-S`, `--https` | Use HTTPS                                       |
 
+For example `gatsby develop --port 4000`.
+
+When running Gatsby CLI commands with options using NPM scripts you need to include `--` to forward the options.
+
+For example `npm run develop -- --port 4000`
+
 Follow the [Local HTTPS guide](https://www.gatsbyjs.org/docs/local-https/)
 to find out how you can set up an HTTPS development server using Gatsby.
 
 ### Build
 
-At the root of a Gatsby site run `gatsby build` to do a production build of a
+At the root of a Gatsby site run `npm run build` to do a production build of a
```

Comment:
I would probably just drop these?

Reply:
Just the line that is changed? Leave the build header and go straight to options after?

Reply:
Yeah - exactly!

Reply:
Same for develop and serve? They have basically the same command between the section heading and the options.

Reply:
```suggestion
At the root of a Gatsby site run `gatsby build` (typically exposed by the NPM script `npm run build`) to do a production build of a
```